operator_ui/src/pages/Jobs/Index.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react'
+import { RouteComponentProps } from 'react-router-dom'
+import Grid from '@material-ui/core/Grid'
+import Button from 'components/Button'
+import { Title } from 'components/Title'
+import Content from 'components/Content'
+import BaseLink from 'components/BaseLink'
+import { v2 } from 'api'
+import * as jsonapi from '@chainlink/json-api-client'
+import * as models from 'core/store/models'
+
+import { TimeAgo } from '@chainlink/styleguide'
+import Card from '@material-ui/core/Card'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableHead from '@material-ui/core/TableHead'
+import TablePagination from '@material-ui/core/TablePagination'
+import TableRow from '@material-ui/core/TableRow'
+import Typography from '@material-ui/core/Typography'
+import TableButtons, { FIRST_PAGE } from 'components/TableButtons'
+import { useHistory } from 'react-router-dom'
+import { formatInitiators } from 'utils/jobSpecInitiators'
+import Link from 'components/Link'
+
+type IndexProps = {
+  pageSize?: number
+} & RouteComponentProps<{
+  pageNumber?: string
+}>
+
+const renderBody = ({
+  error,
+  jobs,
+}: {
+  error?: Error
+  jobs?: jsonapi.PaginatedApiResponse<models.JobSpec[]>['data']
+}) => {
+  if (error) {
+    return (
+      <TableRow>
+        <TableCell component="th" scope="row" colSpan={3}>
+          {error}
+        </TableCell>
+      </TableRow>
+    )
+  } else if (jobs && jobs.length === 0) {
+    return (
+      <TableRow>
+        <TableCell component="th" scope="row" colSpan={3}>
+          You haven’t created any jobs yet. Create a new job{' '}
+          <Link href={`/jobs/new`}>here</Link>
+        </TableCell>
+      </TableRow>
+    )
+  } else if (jobs) {
+    return jobs.map((j) => (
+      <TableRow key={j.id}>
+        <TableCell component="th" scope="row">
+          <Link href={`/jobs/${j.id}`}>{j.id}</Link>
+        </TableCell>
+        <TableCell>
+          <Typography variant="body1">
+            <TimeAgo tooltip>{j.attributes.createdAt || ''}</TimeAgo>
+          </Typography>
+        </TableCell>
+        <TableCell>
+          <Typography variant="body1">
+            {formatInitiators(j.attributes.initiators)}
+          </Typography>
+        </TableCell>
+      </TableRow>
+    ))
+  }
+
+  return (
+    <TableRow>
+      <TableCell component="th" scope="row" colSpan={3}>
+        Loading...
+      </TableCell>
+    </TableRow>
+  )
+}
+
+export const JobsIndex = ({ pageSize = 10, match }: IndexProps) => {
+  React.useEffect(() => {
+    document.title = 'Jobs'
```

Comment:
I believe this will update `document.title` on every re-render. Please add `[]` as second param.